check.py

```diff
@@ -224,6 +224,54 @@ def do_asm2wasm_test():
 run_command(ASM2WASM + [asmjs, '-o', 'b.wast', '-S'])
 assert open('b.wast', 'rb').read()[0] != '\0', 'we emit text with -S'
 
+''' wasm2asm tests disabled for now. Use scripts/test/wasm2asm.py to test
```

Comment:
Better idea: have `check.py` `import scripts.test.wasm2asm as wasm2asm`, wrap the check code in a function, `check_wasm2asm`, and call it from here. (Then comment out the call)